tests/tests_ethtool_ring_initscripts.yml

```diff
@@ -4,10 +4,14 @@
 - hosts: all
   name: Run playbook 'playbooks/tests_ethtool_ring.yml' with initscripts as provider
   tasks:
+    - include_tasks: tasks/el_repo_setup.yml
     - name: Set network provider to 'initscripts'
       set_fact:
         network_provider: initscripts
       tags:
         - always
 
 - import_playbook: playbooks/tests_ethtool_ring.yml
+  when: ansible_distribution in ['CentOS','RedHat'] and
+    ansible_distribution_major_version | int < 9 or
+    ansible_distribution == 'Fedora'
```

Comment:
Please group the conditions with parentheses

Reply:
grouped